<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmCodeCopy</class>
 <widget class="QWidget" name="frmCodeCopy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labPathDemo">
       <property name="text">
        <string>目标路径</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labPathDesigner">
       <property name="text">
        <string>源头路径</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="btnDelete">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>清理文件</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="btnSelectDesigner">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>选择目录</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QLineEdit" name="txtPathInclude">
       <property name="text">
        <string>H:/codeniubi/quc/designer/include</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="btnCopy">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>发布文件</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labPathInclude">
       <property name="text">
        <string>发布路径</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>0</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="btnDo">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>一键搞定</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLineEdit" name="txtPathDesigner">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>3</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>H:/codeniubi/quc/designer</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="btnSelectTarget">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>选择目录</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QComboBox" name="cboxPathTarget">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>H:/codeniubi/quc/demo</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>H:/codeniubi/qui</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>H:/codemodule</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>H:/codevideo</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>H:/project1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>H:/project2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>I:/gitee/QWidgetDemo</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>I:/github/QWidgetDemo</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="txtHeader">
     <property name="undoRedoEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="txtSource">
     <property name="undoRedoEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="txtResult">
     <property name="undoRedoEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>btnSelectTarget</tabstop>
  <tabstop>txtPathDesigner</tabstop>
  <tabstop>btnSelectDesigner</tabstop>
  <tabstop>txtPathInclude</tabstop>
  <tabstop>btnCopy</tabstop>
  <tabstop>btnDelete</tabstop>
  <tabstop>btnDo</tabstop>
  <tabstop>txtHeader</tabstop>
  <tabstop>txtSource</tabstop>
  <tabstop>txtResult</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
